fluid-constructs/src/main/java/me/escoffier/fluid/constructs/CommonHeaders.java

```diff
@@ -0,0 +1,30 @@
+package me.escoffier.fluid.constructs;
+
+import java.util.Optional;
+
+public final class CommonHeaders {
+
+  public static final String KEY = "fluid.key";
+
+  public static final String ORIGINAL = "fluid.original";
```

Comment:
What do you plan to put in _original_ ?

Reply:
nevermind, got it a few line below.

Reply:
Yes, "original" would be an original message from an underlying transport technology.